### PR TITLE
Fix Unit Tests for API 30

### PR DESCRIPTION
**Pull request by david-allison**
Couple of commits split from https://github.com/ankidroid/Anki-Android/pull/13261

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldControllerTest.kt [hunk 1]
```diff
@@ -93,6 +93,8 @@ open class BasicImageFieldControllerTest : MultimediaEditFieldActivityTestBase()
 
     @Test
     fun invalidImageResultDoesNotCrashController() {
+        // TODO: This started failing: showThemedToast threw an NPE. Diagnose the underlying issue.
```
Comment (BrayanDSO) [inline AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldControllerTest.kt:96]: `... started failing after targetSdk 30: ...`

**Review by BrayanDSO — COMMENTED** [review]
Only a nit. Consider this an approval after it

File: AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldControllerTest.kt (hunk 1 above)
Comment (david-allison) [reply AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldControllerTest.kt:96]: I don't have a link for the Robolectric bug. The one mentioned in https://github.com/ankidroid/Anki-Android/pull/13261 was for `isExternalStorageManager()`

Comment (BrayanDSO) [reply AnkiDroid/src/test/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldControllerTest.kt]: Oops. `... started failing after targetSdk 30: ...` then

**Review by BrayanDSO — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
LGTM

**Pull request merged**